docker-php-scripts/docker-php-source

```diff
@@ -14,7 +14,7 @@ usage() {
 	echo
 }
 
-PHP_VER=$(php -v | awk 'NR==1{print $2}')
+PHP_VER=$(php -v | xargs | awk -F 'PHP ' '{print $2}' | awk '{print $1}')
```

Comment:
Hey @alexeylivadnyi - thanks for looking into this. I don't even think it's necessary to parse anything from the `php-v` command since the PHP_VERSION is already defined in the Dockerfile and persisted as an environment variable.

What I would do is remove line 17 completely and change $PHP_VER to $PHP_VERSION.

Reply:
thnx @JalexChen for your help
I've pushed your idea to this PR

Reply:
hey @alexeylivadnyi - tested this and the variable was not available like i thought it would be. however, i did test your original change and rebased. the build is running now. 

i apologize for the extra steps and appreciate you contributing.

@felicianotech can you take a look as well?

Reply:
@JalexChen I've checked variable inside the container - it is exists. Or I've not understand you correctly?
<img width="1227" alt="Снимок экрана 2022-11-07 в 19 02 54" src="https://user-images.githubusercontent.com/108271392/200382858-bd92b5ad-1f96-4e47-ad8c-815126a0aee7.png">

Reply:
@alexeylivadnyi it exists, but not within the scope of the script or function when it's being called via `sudo docker-php-source extract`

Reply:
yes, I see, thanks for the answer